### ISSUE TITLE
Expose client stats of records/bytes transfered

### ISSUE THREAD

**Issue by nemanja-boric-sociomantic** [sociomantic-tsunami/dlsproto#28]
Client should allow inspecting the number of bytes/records transferred for each connection.

**Comment by gavin-norman-sociomantic** [comment]
The number of bytes transmitted per connection is already available via https://github.com/sociomantic-tsunami/swarm/blob/v4.x.x/src/swarm/neo/client/mixins/ClientCore.d#L393-L430. This might be sufficient. (I'm not sure what the use case for this issue is.)

Example of use: https://github.com/sociomantic-tsunami/dhtproto/blob/neo/src/dhtproto/client/UsageExamples.d#L689-L698

**Comment by nemanja-boric-sociomantic** [comment]
Yes, I think that's sufficient. I've opened the issue, as I'm not sure that dlsclient currently is exposing this functionality.

**Comment by gavin-norman-sociomantic** [comment]
I think it's built into the client, but yeah, have a check.
